easy-to-use-api/klips-timelapse-video-api/package.json

```diff
@@ -22,13 +22,13 @@
     "@testing-library/dom": "^9.3.3",
     "@testing-library/jest-dom": "^6.1.4",
     "@types/jest": "^29.5.10",
-    "@typescript-eslint/eslint-plugin": "^6.13.1",
+    "@typescript-eslint/eslint-plugin":"5.62.0",
```

Comment:
Why did you need to downgrade `eslint-plugin` and `typescript`?

Reply:
good question

Reply:
There was a dependency conflict with the newer version: 

> Could not resolve dependency:
> 3.303 npm ERR! dev @typescript-eslint/eslint-plugin@"^6.13.1" from the root project